### DISCUSSION TITLE
git push error

### DISCUSSION THREAD

**Discussion by okdhryk** [hsr-project/general#6] [Q&A]
When I pushed to the scoring server, the following error message was displayed.
It seems that the scoring request was successfully accepted.
It seems to be an error related to VSCODE, what should I do?

Hiroyuki

2021-04-17 00:07:57 ⌚  7a3033f9c80a in /workspace/robocup-at-home-2021-challenge
± |main ↑1 ✓| → git push
fatal: cannot run /home/developer/.theia/extensions/vscode.git-1.52.1/extension/dist/askpass.sh: No such file or directory
Username for 'https://github.com': okdhryk
fatal: cannot run /home/developer/.theia/extensions/vscode.git-1.52.1/extension/dist/askpass.sh: No such file or directory
Password for 'https://okdhryk@github.com': 
Counting objects: 5, done.
Delta compression using up to 2 threads.
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 5.25 KiB | 447.00 KiB/s, done.
Total 5 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To https://github.com/okdhryk/robocup-at-home-2021-challenge.git
   b77b262..99f6370  main -> main

**Comment by yosuke** [comment]
Maybe related:
https://github.com/microsoft/vscode/issues/85569

It seems the actual `git push` is succeeded and the error has only occurred on the vscode side, so we can ignore the error for now. But it is probably better to update the vscode-git extension to fix the problem.
I will return back once I have updated the workspace container image.

**Reply by yosuke** [reply]
It seems the `askpass.sh` is a helper script to store git credentials to vscode to reduce repetitive typing of the password. Would be nice if we can enable the function without error.
